aws/cloudformation/cloud_formation_stack.yml.erb

```
@@ -401,6 +405,45 @@ Resources:
 
 <%   if cdn_enabled && app == 'Pegasus' -%>
 <%=    component 'www_redirect', app: app -%>
+  MarketingRouterLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Description: 'Lambda for dynamically routing marketing traffic to different origins based on the URL path'
+      FunctionName: !Sub "${AWS::StackName}-marketing-router"
+      Code: <%=js 'marketing-router'%>
+      Handler: index.handler
+      Runtime: nodejs22.x
+      Role: !GetAtt MarketingRouterLambdaRole.Arn
+      Environment:
+        Variables:
+          MARKETING_DOMAIN: !Sub "${MarketingDomainName}"
```

If we don't need to do any string interpolation, can we just access the Parameter value directly?
```suggestion
          MARKETING_DOMAIN: !Ref MarketingDomainName
```